### DISCUSSION TITLE
First Illegal Drug Chains

### DISCUSSION THREAD

**Discussion by jasper-one** [EveronLife/EveronLife#130] [Ideas]
I wanted to propose 3 different illegal drug chains that could be implemented to allow Server Owners to get up and running with some illegal activities out of the box. Looking forward to your thoughts, I'm trying to use existing assets where possible.

### **Mushrooms** - Entry Level
****
**Gathering**
There is an existing asset for mushrooms which is good, but I propose re-texturing them to make it more obvious that these are the "illegal" ones. As an entry-level option, these could be as simple as picking apples and players could take them directly to the drug dealer for sale. 
![image](https://user-images.githubusercontent.com/106525617/174505111-aafe7b1d-1f10-46c9-bd6d-7c09b7a3e9cc.png)

To make it somewhat difficult the mushrooms could be placed in one of the many caves around the map. This would require players to offroad and potentially hike to the mushroom gathering spot. 
![image](https://user-images.githubusercontent.com/106525617/174505696-fc7866ba-86da-482c-9c22-322bd3d4c008.png)

### **Heroin** - Standard
****
**Gathering**
Heroin would be a standard process where you gather opium from the Poppy Field. A wide-open space that's relatively easy to access but gives players little protection from rival illegal farmers. 
![image](https://user-images.githubusercontent.com/106525617/174504999-75649365-675e-4f5c-8b35-7e40d2004b1f.png)

I couldn't find a good existing asset so this is something that would probably need to be created. @12gabriel3 if this goes ahead, maybe you could help with the creation of a gathering asset? 
![image](https://user-images.githubusercontent.com/106525617/174505458-e97cd702-0a86-4ae3-a9eb-3b6763d406db.png)

The opium would then be processed into heroin as standard. This could be a lab set up in an "undisclosed" barn. 
![image](https://user-images.githubusercontent.com/106525617/174506388-f73a2247-314b-4453-8e5b-119307ea1699.png)

Another item that would be required for this as well as the below Cocaine chain would be a drug package of sorts that could be recolored to represent the different drugs. 
![image](https://user-images.githubusercontent.com/106525617/174507669-20ea2673-4486-4b69-8e61-a030de1dc9e3.png)

### **Cocaine** - Advanced
****
**Gathering**

Coca Leaves could be gathered from deep in the forest.
![image](https://user-images.githubusercontent.com/106525617/174504979-587efd45-1332-4200-a227-8c8f9029f786.png)
There's a great existing asset for this that can be used (rosa_canina). 
![image](https://user-images.githubusercontent.com/106525617/174506060-ff1a9de7-00c2-45b0-96e6-1bfa90f7d9f4.png)

Diesel could be purchased from a trader at a gas station once #8  is in, at least in a basic state. 
![image](https://user-images.githubusercontent.com/106525617/174506281-46356cce-9ccb-4b66-bcb5-65145d2e1ad5.png)

Cement can be gathered once we sort out #129 
![image](https://user-images.githubusercontent.com/106525617/174505867-109c293f-a377-46e7-bc55-f4d06f00e274.png)

**Procesing**
All of the gathered materials (multi-input) could be combined at a jungle lab (much like the ones in Columbia). This could be processed in a barrel. 
![image](https://user-images.githubusercontent.com/106525617/174506985-3e34c856-62ea-41ed-a19f-940c5cb28ffc.png)

![image](https://user-images.githubusercontent.com/106525617/174505818-50ebb0bc-8d26-4c2c-8c6a-dd9aa70dd154.png)

Keen for all of your thoughts on these "base" illegal professions and any other related ideas.

**Comment by Arkensor** [comment]
I think you already posted some of this in the showcase as well? ![image](https://user-images.githubusercontent.com/8494013/179175631-41462a2b-d692-4e68-838c-467aebb941a6.png)

Sounds like a good idea to me. What was your idea for INPUT > OUTPUT stepts? What do players use in which stage? How many stages are there?

**Comment by jasper-one** [comment]
@Arkensor What is proposed here for Cocaine specifically is a one-process step, 3 inputs. As it stands the process would be: 

The player would gather the following items:
- Coca Leaves (illegal item) from an area with potted cocoa trees. E.g. this could be a few planted around in the depths of the jungle. 
- Diesel Jerry - this would be purchased from a trader (probably the petrol stations) or could potentially be stolen. 
- Cement Bags - gathered as per #90.

At the moment I propose a single process step at maybe a hard-to-reach processor (in the jungle). I would also be fine with multiple process steps at the same physical location. I wanted to keep it to one process location as by this point the player has already spent a lot of time gathering the required input resources from ~3 different locations. 

I'll be cleaning all 3 of the drug chains up over the next few days, so I hope to have a PR where this can all be tested up soon :)
